README.md

```diff
@@ -84,7 +87,7 @@ sudo apt-get install jibri
 
 
 ### User, group
-* Jibri is currently meant to be run as a regular system user. This example creatively uses username 'jibri' and group name 'jibri', but any user will do. This has not been tested with the root user.
+* Jibri is currently meant to be run as a regular system user. This example creatively uses username 'jibri' and group name 'jibri', but any user will do. This has been tested also with the root user.
```

Comment:
I would NOT recommend running jibri (or any daemon) as the root user.  We should explicitly suggest they do NOT do this, as we're not doing anything to ensure that jibri as root would be secure.